test/e2e/run.sh

```diff
@@ -297,6 +288,29 @@ cpu_ids = lambda i: set_ids(i, '[cpu]')
 "
 }
 
+get-res-allowed() {
+    local res_allowed_file="$1"
+    local retries=5
+    while (( retries > 0 )); do
```

Comment:
Thanks @askervin! Looks good. One suggestion: could you add a comment above the retry-loop here rationalizing why retries are needed here (ie.g. something similar than the commit message)